Client/Application/AppDelegate.swift

```diff
@@ -99,6 +99,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject : AnyObject]?) -> Bool {
+        AdjustIntegration.sharedInstance.triggerApplicationDidFinishLaunchingWithOptions(launchOptions)
```

Comment:
Can we kick this to a background queue? Otherwise we have a file stat on the main thread.